pkg/utils/logger.go

```diff
@@ -67,11 +72,19 @@ func (l *logHandle) Log(args ...interface{}) {
 }
 
 func newLogger(name string) *logHandle {
-	l := &logHandle{name: name}
-	l.Out = os.Stderr
+	l := &logHandle{Logger: *logrus.New(), name: name, tty: isatty.IsTerminal(os.Stderr.Fd())}
+	if l.tty {
+		l.prefixes = make([]string, 7)
+		ps := []string{"PANIC", "FATAL", "ERROR", " WARN", " INFO", "DEBUG", "TRACE"}
+		for i := 0; i < 3; i++ {
+			l.prefixes[i] = fmt.Sprintf("\033[1;%dm%s\033[0m", 31, ps[i])
+		}
+		for i := 3; i < 6; i++ {
+			l.prefixes[i] = fmt.Sprintf("\033[1;%dm%s\033[0m", 30+i, ps[i])
+		}
+		l.prefixes[6] = fmt.Sprintf("\033[1;%dm%s\033[0m", 35, ps[6])
```

Comment:
Should we provide a flag to disable this?

Reply:
Better do it; but it may be conflicted with flags of subcommands, like:
```
./juicefs stats -h

   --nocolor          disable colors (default: false)
```

Reply:
we can use it as a global option.

Reply:
Workable